src/components/alert/alert.stories.svelte

```diff
@@ -20,7 +20,7 @@
     showAlert(
       {
         content,
-        title,
+        title: title || undefined,
```

Comment:
This just makes it easier to verify the behavior in Storybook since it doesn't pass a value if the string is empty.